marienfressinaud/FreshRSS
# Install routine: Reload button if checks fail

Closes #964

Changes proposed in this pull request:
- added a reload button in the install routine step 2
![grafik](https://user-images.githubusercontent.com/1645099/129492997-701b4ea9-d0a8-454e-a2a6-b7ebe68f3187.png)


- translation made with Google translate + check with my knowledge + recheck with invers translation

How to test the feature manually:

1. install the app
2. see step 2, when some checks fail


Pull request checklist:
